### PR TITLE
Squad v2

### PR DESCRIPTION
**Pull request by lintangsutawika**
Added 7 prompt templates for squad v2

### CODE REVIEW

**Review by VictorSanh — CHANGES_REQUESTED** [review]
Thank you @lintangsutawika , these look great!
Made a few suggestions.
Could you add a few more templates with more variety in the way you query the model? we posted some recs [here](https://huggingface.slack.com/archives/C023Z96L7B4/p1623244601086900) this morning :)
For instance, this could be something like:
```jinja
My daughter is asking me something about {{title}}: {{question}}.
Here's what I found on the internet: {{context}}.
What is the answer?|||
{{answer.text[0]}}
```

**Comment by lintangsutawika** [comment]
Noted, will add more prompts!

**Comment by lintangsutawika** [comment]
@VictorSanh please check.

File: templates/squad_v2/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+      Context: {{context}};
+
+      Question: {{question}};
+
+      Answer: |||
+
+      {{answers.text[0]}}
+
+      {% endif %}'
```
Comment (VictorSanh) [inline templates/squad_v2/templates.yaml:37]: For easier read, let's factorize this: the prompt is always the same so let's move it outside of the "label" and keep the if condition `{% if answers.text == [] %}` only for the label part -> you won't have two `|||` which you almost never need

File: templates/squad_v2/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+      Answer not in context
+
+      {% else %}
+
+      {{context}} {{ seq | random }}{{question}}|||
+
+      {{answers.text[0]}}
+
+      {% endif %}'
```
Comment (VictorSanh) [inline templates/squad_v2/templates.yaml:178]: same comment as before. factorize the prompt outside of the if condition

File: templates/squad_v2/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+    name: Topic Prediction - Question and Answer Pair
+    reference: Given a Question-Answer pair, generate the topic.
+  fdcf132e-6c70-4188-999e-93601ee8e089: !Template
+    id: fdcf132e-6c70-4188-999e-93601ee8e089
+    jinja: 'What is the follow passage about?
+
+      {{context}} |||
+
+      {{title | replace("_", " ")}}'
+    name: Topic Prediction - Context
```
Comment (VictorSanh) [inline templates/squad_v2/templates.yaml:223]: this makes me think that we'll need to deduplicate inside a training set

**Comment by VictorSanh** [comment]
> @VictorSanh please check.

thank you, small comment on readability
please ping me when you are done, I'll merge then

**Comment by lintangsutawika** [comment]
Hi @VictorSanh have just factorized the templates with conditionals.

File: templates/squad_v2/templates.yaml (hunk 3 above)
Comment (lintangsutawika) [reply templates/squad_v2/templates.yaml:214]: I'd be happy to help write any implementation for the duplication.

**Review by VictorSanh — APPROVED** [review]
thank you merging now, please update the spreadhseet!

**Pull request merged**